### DISCUSSION TITLE
Use Coursera certificate for student developer pack

### DISCUSSION THREAD

**Discussion by AmrShaar13006** [community/community#22121] [New to GitHub]
<p>Can I use my Coursera certificate in my request for a student developer pack?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome to the Community!</p>
<p>The Student Developer Pack is available to students enrolled in degree or diploma granting courses of study, or where an informal learning institution has signed up to <a href="https://education.github.com/schools">GitHub Campus</a>.</p>
<p>Many of the courses available on websites like Coursera would fall under “informal learning” which would not make a student eligible for the pack.</p>

**Comment by AmrShaar13006** [comment]
<p><strong>thank you for replying</strong></p>
<p>Well if I was enrolled in a course offered by the University of London at the Coursera web site, can I benefit from the student developer pack?</p>
<p>Actually, I am in middle school, But I enrolled<br>
on the Coursera website, my school is in Saudi Arabia, and my school does not provide an email to register for the student developer pack,</p>
<p>And the last certificate I got it from my school was the previous year due to the Coronavirus and I am currently studying web design, and I hope to get a student developer pack to benefit from it, so, what can I do?</p>

**Comment by canuckjacq** [comment]
<p>If you are currently enrolled in full time school (even middle school) you can apply. If you ask your school for some kind of proof of enrollment, like a transcript on official headed paper, you can apply since you are a full-time student. You’ll be able to upload proof of enrollment when you apply.</p>

**Reply by johnroha** [reply]
So how do i upload the proof

**Reply by JuanGidoni** [reply]
> So how do i upload the proof

You can read the full article about how to apply to the [Github Student Developer Pack](https://education.github.com/).

At [step 7](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student#applying-to-github-education) you will be asked to upload the proof document/picture. Cheers 🎉
